### PR TITLE
Finished enchanting (I think so)

### PR DESCRIPTION
**Pull request by Glorf**
Here is last part of enchanting.
Some values could be still other than in Morrowind, but I haven't found any problems yet.
As for me, this feature seems to be done now.

### CODE REVIEW

File: apps/openmw/mwmechanics/enchanting.cpp [hunk 1]
```diff
@@ -53,7 +54,12 @@
     {
         ESM::Enchantment enchantment;
         enchantment.mData.mCharge = getGemCharge();
-        mSoulGemPtr.getRefData().setCount (mSoulGemPtr.getRefData().getCount()-1);
+
+        //Exception for Azura Star, it's not destroyed after enchanting
+        if(mSoulGemPtr.get<ESM::Miscellaneous>()->mBase->mId=="Misc_SoulGem_Azura")
```
Comment (ghost) [inline apps/openmw/mwmechanics/enchanting.cpp]: Shouldn't this check be case insensitive?

Comment (Glorf) [inline apps/openmw/mwmechanics/enchanting.cpp]: Ok, fixed

**Comment by Glorf** [comment]
Hmmm, I've checked in ConstructionSet before and azura star had no script attached to it, that's why I thought it's handled by runtime. 
How can I apply this script? I'm not experienced with scripting.

**Comment by ghost** [comment]
Please see mwmechanics/repair.cpp for example.

```
    std::string script = MWWorld::Class::get(itemToRepair).getScript(itemToRepair);
    if(script != "")
        itemToRepair.getRefData().getLocals().setVarByInt(script, "onpcrepair", 1);
```

**Comment by Glorf** [comment]
Hmmm, ok I'll change that

EDIT:  PotatoesMaster checked it, and this script really seem to be unused in Morrowind. So I'll keep it as it is now, until we'll discover how it works.

**Pull request merged**